ci-operator/step-registry/aws/provision/route53/private-hosted-zone/aws-provision-route53-private-hosted-zone-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+REGION="${LEASED_RESOURCE}"
+CLUSTER_NAME="${NAMESPACE}-${JOB_NAME_HASH}"
+
+ROUTE53_HOSTED_ZONE_NAME="${CLUSTER_NAME}.${BASE_DOMAIN}"
+VPC_ID=$(cat "${SHARED_DIR}/vpc_id")
+CALLER_REFERENCE_STR=$ROUTE53_HOSTED_ZONE_NAME
+OUTPUT_ROUTE53_JSON=${SHARED_DIR}/route53_hosted_private_zone.json
```

Comment:
nit: why put this in `SHARED_DIR`?  It doesn't seem to have any consumers at the moment, and space in `SHARED_DIR` is limited.

Reply:
Okay, changes have been implemented.